packages/expo-notifications/src/scheduleNotificationAsync.ts

```diff
@@ -29,14 +29,42 @@ function parseTrigger(userFacingTrigger: NotificationTriggerInput): NativeNotifi
     return { type: 'date', timestamp: userFacingTrigger.getTime() };
   } else if (typeof userFacingTrigger === 'number') {
     return { type: 'date', timestamp: userFacingTrigger };
-  } else if ('seconds' in userFacingTrigger) {
+  } else if (
+    Object.keys(userFacingTrigger).length === 2 &&
+    'hour' in userFacingTrigger &&
+    'minute' in userFacingTrigger
+  ) {
+    const hour = userFacingTrigger.hour;
+    const minute = userFacingTrigger.minute;
+    if (hour === undefined || minute === undefined) {
+      throw new TypeError('Both hour and minute need to have valid values. Found undefined');
+    }
+    if (hour < 0 || hour > 23) {
+      throw new RangeError(`The hour parameter needs to be between 0 and 23. Found: ${hour}`);
+    }
+    if (minute < 0 || minute > 59) {
+      throw new RangeError(`The minute parameter needs to be between 0 and 59. Found: ${minute}`);
+    }
+    return {
+      type: 'daily',
+      hour,
+      minute,
+    };
+  } else if (
+    (Object.keys(userFacingTrigger).length === 1 && 'seconds' in userFacingTrigger) ||
+    (Object.keys(userFacingTrigger).length === 2 &&
+      'seconds' in userFacingTrigger &&
+      'repeats' in userFacingTrigger)
+  ) {
```

Comment:
The unit tests exposed that a calendar trigger with seconds would get incorrectly identified as a timeInterval.

Reply:
Oh I see. I did miss that. I removed the above as correcting the test takes care of it. 

Hmm. I don't think I can add a check / TypeError that you have in #8261 for the dailyTrigger. However there could be a conflict between having minute and hour set and expecting repeats to default to false. The parameters would have to be changed to `minutes` / `hours` in order to properly check for this but